Makefile

```diff
@@ -127,17 +134,19 @@ lint: golangci-lint ## Run golangci-lint against code.
 
 ##@ Build
 
-.PHONY: manager
-manager: ## Build manager binary.
-	go build -o $@
+manager: $(shell find -name "*.go") go.mod go.sum  ## Build manager binary.
+	go build -o $@ ./cmd/manager
+
+manager-hub: $(shell find -name "*.go") go.mod go.sum  ## Build manager-hub binary.
+	go build -o $@ ./cmd/manager-hub
 
 .PHONY: run
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
 .PHONY: docker-build
-docker-build: unit-test ## Build docker image with the manager.
-	docker build -t $(IMG) .
+docker-build: ## Build docker image with the manager.
```

Comment:
why we no longer want to run unit-test for docker-build

Reply:
It slowed my development workflow and did not bring any benefit. Do you insist on keeping it?

Reply:
i think it is a good benchguard. I don't mind that it will be moved to a different target, as long as that target will be called during the image build by CI.

Reply:
Running tests is quite different from building the image though. It involves downloading `controller-gen` because we may need to regenerate mocks.
Neither upstream nor midstream use the target to build images for that reason. But the unit tests are ran in a different job.

Reply:
+1 for removing it.

UT are tested in CI regardless of building the image so we have no risk here.